lib/rdoc/options.rb

```diff
@@ -989,6 +998,13 @@ def parse argv
 
       opt.separator nil
 
+      opt.on("--autolink-excluded-words=WORDS", Array,
+             "Words to be ignored in autolink cross-references") do |value|
+        @autolink_excluded_words = value
```

Comment:
To give this options multiple times:

```suggestion
        @autolink_excluded_words.concat value
```

Reply:
Updated 👍